processing/src/main/java/io/druid/query/metadata/metadata/SegmentMetadataQuery.java

```diff
@@ -98,20 +138,32 @@ public String getType()
         getQuerySegmentSpec(),
         toInclude,
         merge,
-        getContext());
+        getContext(),
+        isUsingDefaultInterval()
+    );
   }
 
   @Override
   public boolean equals(Object o)
```

Comment:
This should include the new field - you can regenerate this and hashCode with IntelliJ